Test/dafny0/SubsetTypes.dfy.expect

```diff
@@ -391,5 +391,13 @@ Execution trace:
 SubsetTypes.dfy(455,9): Error: value does not satisfy the subset constraints of 'nat'
 Execution trace:
     (0,0): anon0
+SubsetTypes.dfy(465,7): Error: cannot find witness that shows type is inhabited (only tried 'D'); try giving a hint through a 'witness' or 'ghost witness' clause
```

Comment:
SubsetTypes.dfy failed CI -- is it still a problem or is the expect file obsolete?

Reply:
I had forgotten to update the `.expect` file in the `allocated1/dafny0` folder. Thanks.